addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -736,11 +738,19 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
     willStart: async function () {
         const _super = this._super.bind(this);
         // Get the authorized existing fields for the form model
-        this.existingFields = await this._rpc({
-            model: "ir.model",
-            method: "get_authorized_fields",
-            args: [this.formEl.dataset.model_name],
-        }).then(fields => {
+        const model = this.formEl.dataset.model_name;
+        var fields;
```

Comment:
I think `let` is more appropriate than `var` here.

Reply:
done https://github.com/odoo/odoo/compare/30159f2bfb6bce76a26ae795a5d5ec1046fca8f7..a09929b286f9472de74e3b650dcb2eb0fbf3fc93

Reply:
Also here https://github.com/odoo/odoo/compare/a09929b286f9472de74e3b650dcb2eb0fbf3fc93..5c759b3b4fab381361dd71d40ed1c8321bfe073f